platform/web/detect.py

```diff
@@ -173,7 +166,7 @@ def configure(env):
 
     # All intermediate files are just LLVM bitcode.
```

Comment:
Nitpick but maybe this comment is no longer accurate given the warning that Emscripten emits with `.bc` extension?
```
em++: error: .bc output file suffix used without -flto or -emit-llvm.  Consider using .o extension since emcc will output an object file, not a bitcode file [-Wemcc] [-Werror]
```